crmsh/bootstrap.py

```diff
@@ -2417,4 +2406,52 @@ def get_stonith_timeout_generally_expected():
         return None
 
     return STONITH_TIMEOUT_DEFAULT + corosync.token_and_consensus_timeout()
+
+
+def adjust_pcmk_delay():
+    """
+    For each fence agent,
+    add parameter pcmk_delay_max when cluster is two-node cluster without qdevice
+    else remove pcmk_delay_max
+    """
+    cib_factory.refresh()
+
+    if utils.is_2node_cluster_without_qdevice():
+        for res in cib_factory.fence_id_list_without_pcmk_delay():
```

Comment:
To make adjust_pcmk_delay() more general even beyond bootstrap, I suggest to adjust only if pcmk_delay_max < PCMK_DELAY_MAX .

Reply:
Seems still no harm to keep it unchanged?:)

Reply:
`fence_id_list_without_pcmk_delay` will show the fence agent id list which don't contain pcmk_delay_max/base, or the value <= 0

Reply:
Agree, just stay within the bootstap scope, it is enough. 

I keep thinking this is a low hanging fruit to go extra miles for `crm cluster health` concept in the near future.